### ISSUE TITLE
Error when trying to run in mainnet

### ISSUE THREAD

**Issue by magick93** [LiskHQ/lisk-sdk#3887]
### Expected behavior

Expect it to not error

### Actual behavior

```

WARNING: Creating a duplicate QueryFile object for the same file -
--
  | /opt/app-root/src/framework/src/components/storage/sql/accounts/get.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Account.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Account (/opt/app-root/src/framework/src/components/storage/entities/account.js:281:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:37:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/accounts/get_extended.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Account.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Account (/opt/app-root/src/framework/src/components/storage/entities/account.js:281:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:37:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/accounts/count.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Account.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Account (/opt/app-root/src/framework/src/components/storage/entities/account.js:281:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:37:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/accounts/is_persisted.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Account.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Account (/opt/app-root/src/framework/src/components/storage/entities/account.js:281:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:37:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/transactions/get.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Transaction.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Transaction (/opt/app-root/src/framework/src/components/storage/entities/transaction.js:245:20)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:165:28)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/transactions/get_extended.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Transaction.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Transaction (/opt/app-root/src/framework/src/components/storage/entities/transaction.js:245:20)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:165:28)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/transactions/is_persisted.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Transaction.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Transaction (/opt/app-root/src/framework/src/components/storage/entities/transaction.js:245:20)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:165:28)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/transactions/count.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Transaction.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Transaction (/opt/app-root/src/framework/src/components/storage/entities/transaction.js:245:20)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:165:28)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/transactions/count_all.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Transaction.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Transaction (/opt/app-root/src/framework/src/components/storage/entities/transaction.js:245:20)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:165:28)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/blocks/get.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Block.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:208:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/blocks/count.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Block.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:208:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate QueryFile object for the same file -
  | /opt/app-root/src/framework/src/components/storage/sql/blocks/is_persisted.sql
  | at PgpAdapter.loadSQLFile (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:173:14)
  | at Object.keys.forEach.key (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:221:56)
  | at Block.loadSQLFiles (/opt/app-root/src/framework/src/components/storage/entities/base_entity.js:219:25)
  | at new Block (/opt/app-root/src/framework/src/components/storage/entities/block.js:208:20)
  | at Storage.registerEntity (/opt/app-root/src/framework/src/components/storage/storage.js:86:24)
  | at createStorageComponent (/opt/app-root/src/framework/src/components/storage/index.js:38:10)
  | at Chain.bootstrap (/opt/app-root/src/framework/src/modules/chain/chain.js:123:20)
  |  
  | WARNING: Creating a duplicate database object for the same connection.
  | at PgpAdapter.connect (/opt/app-root/src/framework/src/components/storage/adapters/pgp_adapter.js:95:18)
  | at Storage.bootstrap (/opt/app-root/src/framework/src/components/storage/storage.js:43:23)
  | at module.exports (/opt/app-root/src/framework/src/modules/http_api/init_steps/bootstrap_storage.js:19:32)
  | at HttpApi.bootstrap (/opt/app-root/src/framework/src/modules/http_api/http_api.js:76:9)
  |  
  | [{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/ReducedBlock","path":["definitions","ReducedBlock"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/CommonBlock","path":["definitions","CommonBlock"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/PeersList","path":["definitions","PeersList"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/parameters/data","path":["parameters","data"]}] 2


```

### Which version(s) does this affect? (Environment, OS, etc...)

Running in Kubernetes (OpenShift 3.11) using node 10.16.0.

**Comment by shuse2** [comment]
@magick93 
Thank you for reporting!
Could you tell us which version of `lisk-sdk` and `lisk-core` you are using for the above error?

**Comment by magick93** [comment]
Hi @shuse2 

I've branched from `v1.6.0` and made this modification - https://github.com/hughestech/lisk-sdk/blob/1.6.0-node-lts/package.json#L18-L21

I think that is the only think I've changed.

**Comment by shuse2** [comment]
@magick93 I think this is a warning that was happening from the internal library on 1.6, but it should not affect the application itself

**Comment by magick93** [comment]
Ok, interesting. 

It does result in the container crashing and restarting.

**Comment by magick93** [comment]
Is there a verbose logging option?

**Comment by magick93** [comment]
When I try to manually start lisk, I get the following output:


```
sh-4.2$ /usr/bin/npm start --network mainnet
npm info it worked if it ends with ok
npm info using npm@6.9.0
npm info using node@v10.16.0
npm info lifecycle lisk@1.6.0~prestart: lisk@1.6.0
npm info lifecycle lisk@1.6.0~start: lisk@1.6.0

> lisk@1.6.0 start /opt/app-root/src
> node src/index.js "mainnet"

[inf] 2019-07-01 02:41:04 | Starting the app - devnet-4000 - Lisk Framework(0.1.0)
[inf] 2019-07-01 02:41:04 | Initializing controller
[inf] 2019-07-01 02:41:04 | Loading controller
[inf] 2019-07-01 02:41:04 | Old PID: 31
[inf] 2019-07-01 02:41:04 | Current PID: 80
[inf] 2019-07-01 02:41:04 | Loading module lisk-core-chain:0.1.0 with alias "chain"
[inf] 2019-07-01 02:41:04 | Module ready with alias: chain(lisk-core-chain:0.1.0)
[inf] 2019-07-01 02:41:04 | Loading module lisk-core-http-api:0.1.0 with alias "httpApi" as child process
[inf] 2019-07-01 02:41:04 | Event httpApi:loading:finished was subscribed but not registered to the bus yet.
[inf] 2019-07-01 02:41:05 | Module ready with alias: httpApi(lisk-core-http-api:0.1.0)
[inf] 2019-07-01 02:41:05 | Bus listening to events - ["lisk:ready","chain:blocks:change","chain:signature:change","chain:transactions:change","chain:rounds:change","chain:multisignatures:signature:change","chain:multisignatures:change","chain:delegates:fork","chain:loader:sync","chain:dapps:change","chain:registeredToBus","chain:loading:started","chain:loading:finished","httpApi:registeredToBus","httpApi:loading:started","httpApi:loading:finished"]
[inf] 2019-07-01 02:41:05 | Bus ready for actions - ["lisk:getComponentConfig","chain:calculateSupply","chain:calculateMilestone","chain:calculateReward","chain:generateDelegateList","chain:updateForgingStatus","chain:getPeers","chain:getPeersCountByFilter","chain:postSignature","chain:getForgersPublicKeys","chain:getTransactionsFromPool","chain:getLastCommit","chain:getBuild","chain:postTransaction","chain:getDelegateBlocksRewards","chain:getSlotNumber","chain:calcSlotRound","chain:getNodeStatus"]
[inf] 2019-07-01 02:41:05 | Releasing enqueued broadcasts
[inf] 2019-07-01 02:41:05 | Queue empty
[ERR] 2019-07-01 02:41:05 | error: duplicate key value violates unique constraint "blocks_height"
    at Connection.parseE (/opt/app-root/src/node_modules/pg/lib/connection.js:554:11)
    at Connection.parseMessage (/opt/app-root/src/node_modules/pg/lib/connection.js:379:19)
    at Socket.<anonymous> (/opt/app-root/src/node_modules/pg/lib/connection.js:119:22)
    at Socket.emit (events.js:198:13)
    at Socket.EventEmitter.emit (domain.js:448:20)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
[ERR] 2019-07-01 02:41:05 | error: current transaction is aborted, commands ignored until end of transaction block
    at Connection.parseE (/opt/app-root/src/node_modules/pg/lib/connection.js:554:11)
    at Connection.parseMessage (/opt/app-root/src/node_modules/pg/lib/connection.js:379:19)
    at Socket.<anonymous> (/opt/app-root/src/node_modules/pg/lib/connection.js:119:22)
    at Socket.emit (events.js:198:13)
    at Socket.EventEmitter.emit (domain.js:448:20)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
[ERR] 2019-07-01 02:41:05 | BatchError: duplicate key value violates unique constraint "blocks_height"
    at check (/opt/app-root/src/node_modules/spex/lib/ext/batch.js:136:32)
    at step (/opt/app-root/src/node_modules/spex/lib/ext/batch.js:110:17)
    at /opt/app-root/src/node_modules/spex/lib/ext/batch.js:85:17
    at /opt/app-root/src/node_modules/spex/lib/utils/index.js:64:25
    at bound (domain.js:402:14)
    at runBound (domain.js:415:12)
    at tryCatcher (/opt/app-root/src/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/opt/app-root/src/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/opt/app-root/src/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/opt/app-root/src/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/opt/app-root/src/node_modules/bluebird/js/release/promise.js:690:18)
    at _drainQueueStep (/opt/app-root/src/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/opt/app-root/src/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/opt/app-root/src/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/opt/app-root/src/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:705:18)
[FTL] 2019-07-01 02:41:05 | Chain initialization - {}
[inf] 2019-07-01 02:41:05 | Cleanup controller...
[ERR] 2019-07-01 02:41:05 | Reason: Blocks#saveBlock error
[ERR] 2019-07-01 02:41:05 | Caused error during cleanup - {}
[inf] 2019-07-01 02:41:05 | Cleanup controller...
npm info lifecycle lisk@1.6.0~poststart: lisk@1.6.0
npm timing npm Completed in 2630ms
npm info ok 


   ╭───────────────────────────────────────────────────────────────╮
   │                                                               │
   │      New patch version of npm available! 6.9.0 -> 6.9.2       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.9.2   │
   │               Run npm install -g npm to update!               │
   │                                                               │
   ╰───────────────────────────────────────────────────────────────╯

sh-4.2$ [{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/ReducedBlock","path":["definitions","ReducedBlock"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/CommonBlock","path":["definitions","CommonBlock"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/definitions/PeersList","path":["definitions","PeersList"]},{"code":"UNUSED_DEFINITION","message":"Definition is not used: #/parameters/data","path":["parameters","data"]}] 2
[inf] 2019-07-01 02:41:05 | Lisk started: 0.0.0.0:4000


```

Interestingly, calling `/usr/bin/npm start --network mainnet` still seems to result in:

```
> node src/index.js "mainnet"

[inf] 2019-07-01 02:41:04 | Starting the app - devnet-4000 - Lisk Framework(0.1.0)
```
So, is it running `mainnet` or `devnet`?

**Comment by pablitovicente** [comment]
Hello @magick93 you need to run the command `node src/index.js --network mainnet` in 1.6 you can run `node src/index.js --help` to check how to pass parameters. 

In your command as it's missing the `mainnet` it's defaulting to devnet.

**Comment by pablitovicente** [comment]
Also if you want to use `npm start` you can do so like this `npm start --network -- --network mainnet`  (notice the extra `--` before the parameters for index.js)

**Comment by magick93** [comment]
Thanks @pablitovicente

**Comment by magick93** [comment]
This is the command I am using `/usr/bin/npm start -- --network mainnet`

And this is what I see in the logs:
```

npm info it worked if it ends with ok
--
  | npm info using npm@6.9.0
  | npm info using node@v10.16.0
  | npm info lifecycle lisk@1.6.0~prestart: lisk@1.6.0
  | npm info lifecycle lisk@1.6.0~start: lisk@1.6.0
  |  
  | > lisk@1.6.0 start /opt/app-root/src
  | > node src/index.js
  |  
  | [inf] 2019-07-01 21:25:50 \| Starting the app - devnet-4000 - Lisk Framework(0.1.0)
  | [inf] 2019-07-01 21:25:50 \| Initializing controller
  | [inf] 2019-07-01 21:25:50 \| Loading controller


```

**Comment by magick93** [comment]
Ok I managed to get it to use `mainnet` by setting the envvar `LISK_NETWORK` to `mainnet`. 
Howvever this was intially a bit confusing as, unlike in `devnet` there was no console logging happening but for the initial warning messages about `WARNING: Creating a duplicate QueryFile object for the same file` - (see https://github.com/LiskHQ/lisk-sdk/issues/3887#issue-461854031).

Once I set `LISK_CONSOLE_LOG_LEVEL` to `info` I was then able to see it was working on `mainnet`.

I couldnt see any cli options to display the current in use network.
